components/controller.ts

```diff
@@ -427,11 +427,13 @@ export class Controller {
     }
 
     /**
-     * Adds a dependency on a client side mod through the Simple Mod Framework.
-     * See the cookbook for a usage example!
+     * You should use `modIsInstalled` instead!
+     *
+     * Returns whether a mod is UNAVAILABLE.
      *
      * @param modId The mod's ID.
-     * @returns If the mod is available. You will probably want to abort plugin initialization if false is returned.
+     * @returns If the mod is unavailable. You should probably abort initialisation if true is returned. Also returns true if the `overrideFrameworkChecks` flag is set.
```

Comment:
```suggestion
     * @returns If the mod is unavailable. You should probably abort initialization if true is returned. Also returns true if the `overrideFrameworkChecks` flag is set.
```
All other docstrings use American English, so this makes sense for consistency's sake.